src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -177,18 +275,80 @@ const CreateCacheWizard = (props) => {
         nextButtonText: 'Create Cache',
     };
 
-    const steps = [stepGettingStarted, stepEditCode];
+    const stepConfigure = {
+        name: 'Configuration',
+        steps: [
+            {
+                id: 3,
+                name: 'Basic',
+                component: (
+                    <ConfigurationBasic formConfigurationBasic={formConfigurationBasic} handleFormConfigurationBasic={handleFormConfigurationBasic} />
+                )
+            },
+            { id: 4, name: 'Cache Feature', component: <p>Cache Feature</p> },
```

Comment:
try to already export these into the translator file. DOn't use <p>, use text content component